src/vm.ts

```diff
@@ -1,5 +1,5 @@
 import { VM } from 'vm2'
-import fetch from 'node-fetch'
+import * as fetch from 'node-fetch'
```

Comment:
What do the introduced asterisks change? 🤔

Reply:
These go together with the `esModuleInterop` change in `tsconfig.json`. If we're writing typescript, might as well write real typescript, and this is what that looks like. Also, everybody including React is moving away from default exports, so we might as well adapt.